src/app/store/reducers/VehicleTestResultModel.reducers.spec.ts

```diff
@@ -53,5 +57,52 @@ describe('VehicleTestResultModel Reducer', () => {
     });
   });
 
-});
+  describe('[SetCurrentState]', () => {
+    test('should update currentState', () => {
+      const action = new SetCurrentState(VIEW_STATE.VIEW_ONLY);
+      const result = VehicleTestResultModelReducers(initialVehicleTestResultModelState, action);
+
+      expect(result).toMatchObject({
+        ...initialVehicleTestResultModelState,
+        editState: VIEW_STATE.VIEW_ONLY
+      });
+    });
+  });
+
+  describe('[Update Test Result]', () => {
+    test('should update testStations', () => {
```

Comment:
adopt a consistent approach

Reply:
use it agree and also needs to be more descriptive please like mentioned above. There is more happening to it.
this will be more legible and intelligible.